autoload/neomake/makers/ft/jsx.vim

```diff
@@ -5,7 +5,7 @@ function! neomake#makers#ft#jsx#SupersetOf()
 endfunction
 
 function! neomake#makers#ft#jsx#EnabledMakers()
-    return ['jshint', 'eslint']
+    return ['jshint', 'eslint', 'eslint_d']
```

Comment:
`jscs` was merged with `eslint`. Maybe, it should be removed as a default from javascript's enabled makers.